documentation/smg/11_ISIS_release.md

```diff
@@ -1,27 +1,43 @@
 # ISIS Release Process
 
-Horace is deployed separately to a number of systems across ISIS
+Horace deployed on iDAAaS system and machines dedicated for excitations as a single git repository available 
+read-only for excitation machine users and providing read/write access for the members of the excitations group.
 
-## ISIS Compute
+As the single place of the Horace code is available to all machines, the changes to the code made by releasers are immediately available to all iDAAaS machines.
 
-#### Contact
+## Horace installation in ISIS
+
+Common path `/usr/local/mprogs/Users` is added to the search path of all MATLAB sessions available to users processing the results of inelastic neutron scattering experiments in ISIS.
+ 
+On the machines, managed by excitation group this is achieved by adding this path to the end of the `{MATLAB Installation Folder}/toolbox/local/pathdef.m` file, though recent iDAAaS virtual machines seems use different approach for adding this path to the MATLAB search path.
```

Comment:
```suggestion
On the machines managed by the Excitations group, this is achieved by adding this path to the end of the list of paths referenced in the`{MATLAB Installation Folder}/toolbox/local/pathdef.m` file. 

Note that recent iDAaaS virtual machines use a different approach for adding this path to the MATLAB search path.
```

There needs to be more detail about the different approach. We can't have an instruction that doesn't work on half the IDAaaS machines.

Reply:
I hondestly do not know what they do but `pathdef.m` is unchanged. Probably they wrap Matlab launch script into into some other script or set up some enviromental variables.